content/pages/sprint/2024/08-coscup.rst

```diff
@@ -0,0 +1,229 @@
+
+In addition, you can also customize your own rules to suit your team. Commitizen-tools is a widely used tool in many communities. We still have a lot of issues to fix and develop. Welcome to join us.
+
+Career Roundtable
+----------------------
+
+Career Roundtable is to have a relaxed discussions with experienced software professionals. 
+At this time, we invited many professional software engineers as our mentor. 
+If you have career-related questions or seek for some guidances, do not miss out this chance!
+ 
+Technical Talks
+-------------------
+
+Packet Party: Getting Down and Dirty with PcapPlusPlus
++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+- **Speaker**: Anchi Liu
+- **About speaker**: 
+    Liu works as a software engineer in Mujin, Inc., Tokyo, Japan. He uses @tigercosmos as the 
+    name in the open-source world, and his Chinese nickname is “微中子.” He holds an M.S. degree in Computer Science 
+    from National Yang Ming Chiao Tung University (NYCU) in 2022, Taiwan. He got a B.S. degree in 
+    Bio-Industrial Mechatronics Engineering from National Taiwan University (NTU) in 2019.
+- **Abstract**:
+    PcapPlusPlus is a cross-platform C++ library capable of packet interception, parsing, and construction. 
+    Additionally, it offers a range of functionalities such as implementing various network protocols, supporting 
+    DPDK, and providing simplified packet filtering. The presentation will cover the principles of PcapPlusPlus 
+    and libpcap, the speaker’s contributions to PcapPlusPlus, and some cases studies. 
+
+|
+
+A High-Efficiency Natural Selection Simulator with Pybind11
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+- **Speaker**: YJack
+- **About speaker**: 
+    我是 YJ，不活躍於各大論壇，組織過最大的東西是跟朋友的讀書會。但是我還是在學校、競賽、專案的過程還是學會了很多東西。
+    所以不管對於我個人或是專案有各種問題都可以來找我聊聊或是討論喔！
+- **Abstract**:
+    This presentation introduces SimEvo, a natural selection simulator with a core engine implemented in C++ 
+    for functionality and algorithms, and a simple Python interface provided via Pybind11. SimEvo allows users 
+    to simulate ecosystems where organisms react, interact, and reproduce based on genetic traits. I will 
+    showcase the project architecture, explaining how computationally intensive parts are handled in C++ and 
+    how the main APIs are ported to Python, enabling users to easily customize different scenarios through 
+    scripting. 
+
+|
+
+探索開源大型語言模型的科學實務應用: RAG 知識管理和 LLM 服務化的作法
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+- **Speaker**: 劉育維 Simon Liu
+- **About speaker**:
+    大家好，我是 Simon 劉育維，過去曾擔任過電信業的資深工程師 和 在軟體業擔任 MLOps 客戶技術成功工程師，
+    幫助各大知名企業進行機器學習、深度學習、大型語言模型等人工智慧議題進行人工智慧架構規劃的討論，目前在 Medium 上已經公開超過 70 
+    篇技術文章，我希望能夠嘗試使用 AI 做應用，幫助客戶用 AI 解決痛點。
+- **Abstract**: 
```

Comment:
![image](https://github.com/user-attachments/assets/60e8f055-d98d-4144-a993-d61afa194756)
I'm not sure what's going on, but there seems to be something wrong with the line spacing here, shown as the above figure. Compared to other talks, this talks has wider line spacing. Do you have any idea what is going on here?

Reply:
I'm wondering why as well, but I think it might be title in Chinese affects, need to test and verify, but I think it is good for this stage.